includes/class-ajax-handlers.php

```diff
@@ -86,7 +86,11 @@ function get_posts_by() {
 			'post_type' => array_keys($this->project_organizer->available_post_types()),
 			'posts_per_page' => -1,
 			'orderby' => 'post_date',
-			'order' => 'DESC'
+			'order' => 'DESC',
+			'post_status' => apply_filters(
+				'anth_included_post_stasuses',
```

Comment:
Could we please change this to 'anthologize_source_item_post_statuses'?

Reply:
:+1: